src/views/Correction.vue

```diff
@@ -975,4 +980,12 @@ h2 {
     margin-left: 0.5rem;
   }
 }
+
+:deep() {
```

Comment:
I always like to see a comment indicating why you need to deep-override some CSS.